modules/ROOT/pages/introduction/cypher_tutorial.adoc

```diff
@@ -699,63 +733,73 @@ For more information about the different label expressions supported by Cypher,
 
 .Query
-[source, cypher, role=test-result-skip]
+[source, cypher]
 ----
-MATCH (tom:Person {name:'Tom Hanks'})-[*2]-(colleagues:Person)
-RETURN colleagues.name as colleagues LIMIT 5
+MATCH (tom:Person {name:'Tom Hanks'})-->{2}(colleagues:Person)
+RETURN DISTINCT colleagues.name AS name, colleagues.born AS bornIn  
+ORDER BY bornIn LIMIT 5
 ----
 
 .Result
-[role="queryresult",options="header,footer",cols="1*<m"]
+[role="queryresult",options="header,footer",cols="2*<m"]
 |===
-|+colleagues+
+| name | bornIn
 
-| +"Parker Posey"+
-| +"Greg Kinnear"+
-| +"Dave Chappelle"+
-| +"Nora Ephron"+
-| +"Meg Ryan"+
+| "Mike Nichols" | 1931
+| "Ian McKellen" | 1939
+| "James Cromwell" | 1940
+| "Nora Ephron" | 1941
+| "Penny Marshall" | 1943
 
-1+d|Rows: 5
+2+d|Rows: 5
 |===
 
 It is also possible to match a graph for patterns of a variable length. 
-The below query matches all `Person` nodes up to 3 "hops" away from `Tom Hanks` and returns the first five rows. 
-The xref:syntax/operators.adoc#syntax-using-the-distinct-operator[DISTINCT] operator ensures that the result contain no duplicate values. 
+The below query matches all `Person` nodes between `1` and `4` "hops" away from `Tom Hanks` and returns the first five rows, ordered by the `born` property value of the `colleagues` nodes in an ascending order.
 
 .Query
 [source, cypher, role=test-result-skip]
 ----
-MATCH (tom:Person {name:'Tom Hanks'})-[*1..3]-(colleagues:Person)
-RETURN DISTINCT colleagues.name as colleagues LIMIT 5
+MATCH (p:Person {name:'Tom Hanks'})-->{1,4}(colleagues:Person)
+RETURN DISTINCT colleagues.name AS name, colleagues.born AS bornIn  
+ORDER BY bornIn LIMIT 5
 ----
 
 .Result
-[role="queryresult",options="header,footer",cols="1*<m"]
+[role="queryresult",options="header,footer",cols=2*<m"]
 |===
-|+colleagues+
+| name | bornIn
 
-| +"David Mitchell"+
-| +"Lana Wachowski"+
-| +"Halle Berry"+
-| +"Hugo Weaving"+
-| +"Jim Broadbent"+
+| "Max von Sydow" | 1929
+| "Richard Harris" | 1930
+| "Clint Eastwood" | 1930
+| "Gene Hackman" | 1930
+| "Mike Nichols" | 1931
 
-1+d|Rows: 5
+2+d|Rows: 5
 |===
 
+[NOTE]
+====
+The quantifier used in the above two examples was introduced with the release of xref::patterns/concepts.adoc#quantified-path-patterns[quantified path patterns] in Neo4j 5.9.
+Before that, the only way in Cypher to match paths of a variable length was with a variable-length relationship.
+This syntax is still available in Cypher. 
+Read more about it xref::patterns/reference.adoc#variable-length-relationships[here].
+====
+
 To find the shortest possible path between two nodes, use the `shortestPath` algorithm.
 For example, this query matches the shorest path in the graph between the two nodes `Tom Hanks` and `Keanu Reeves`:
 
 .Query
 [source, cypher]
 ----
 MATCH p=shortestPath(
-(keanu:Person {name:"Keanu Reeves"})-[*]-(tom:Person {name:"Tom Hanks"})
+(keanu:Person {name:"Keanu Reeves"})-[*]->(tom:Person {name:"Tom Hanks"})
```

Comment:
Are you sure about adding the direction? The diagram shows alternating relationship directions, so I can't see how that pattern would match.

Reply:
No, you are correct. Thanks!